exercise-src/react-native/06-custom-components/03-problem/src/App.tsx

```diff
@@ -22,13 +24,20 @@ export const ListItem: React.FC<ListItemProps> = ({ name }) => {
 
 export const StateList: React.FC = () => {
   return (
-    <View>
-      {states?.length ? (
-        states.map((state) => <ListItem key={state.short} name={state.name} />)
-      ) : (
-        <Text>No states found</Text>
-      )}
-    </View>
+    <ScrollView>
```

Comment:
In Navigation, I had to add `noScroll` to fix this error: `VirtualizedLists should never be nested inside plain ScrollViews with the same orientation because it can break windowing and other functionality - use another VirtualizedList-backed container instead.`

I wonder if we’ll need to do the same here.

Reply:
NoScroll is only needed when using our `Screen` component (which scrolls by default) when there's a `FlatList` rendering inside of it. This has just `View`s right now, so its safe inside `ScrollView`.